acquisitions/src/main/resources/karate-config.js

```diff
@@ -105,13 +105,23 @@ function fn() {
       password: 'admin'
     }
   } else if (env == 'rancher') {
-        config.baseUrl = 'https://thunderjet-okapi.ci.folio.org';
-        config.edgeUrl = 'https://folio-snapshot.dev.folio.org:8000';
-        config.admin = {
-          tenant: 'diku',
-          name: 'diku_admin',
-          password: 'admin'
-        }
+    config.baseUrl = 'https://thunderjet-okapi.ci.folio.org';
+    config.edgeUrl = 'https://folio-snapshot.dev.folio.org:8000';
+    config.admin = {
+      tenant: 'diku',
+      name: 'diku_admin',
+      password: 'admin'
+    }
+  } else if(env == 'folio-testing-karate'){
```

Comment:
Please can consistent formatting be used for the braces in these statements.

Some separate the end opening brace from the rest of the statement and others do not.

```suggestion
  } else if(env == 'folio-testing-karate') {
```

Reply:
done